ingress-examples/README.md

```diff
@@ -14,5 +14,20 @@
 
 [ingress-subdir-non-root.yaml](ingress-subdir-non-root.yaml): use your.com/kubecost instead of kubecost on the root path
 
+## Quickstart if using Nginx Ingress Controller (`ingressClassName: nginx`)
 
+```sh
+# 1. Nginx helm chart
+$ helm repo add nginx-stable https://helm.nginx.com/stable
+$ helm repo update
+$ helm install nginx nginx-stable/nginx-ingress -n kube-system
```

Comment:
it should be installed in a separate namespace than `kube-system` . Maybe something like this:

```bash
$ helm repo add nginx-stable https://helm.nginx.com/stable
$ helm repo update
$ export NAMESPACE="nginx-ingress" # update with your desired namespace
$ helm install nginx nginx-stable/nginx-ingress -n $NAMESPACE --create-namespace
```

Reply:
nginx doesn't maintain ingress-nginx. This is the docs for ingress-nginx https://kubernetes.github.io/ingress-nginx/

which is very confusing. 

NGINX maintains nginx-ingress and nginx-plus, which we don't use.

Reply:
Agree with both that the ingress controller should be in a separate namespace.

I'll update the docs with this command:

```
helm install ingress-nginx ingress-nginx \
  --repo https://kubernetes.github.io/ingress-nginx \
  --namespace ingress-nginx
```

And this doc:

https://docs.nginx.com/nginx-ingress-controller/installation/installation-with-helm/